templates/view.html

```diff
@@ -75,10 +75,10 @@
                 <b>{{ call $.T "report_type" }}:</b>
                 <form method="post" action="/report/{{$.Torrent.ID}}">
                     {{ block "csrf_field" $ }}{{end}}
-                    <input type="radio" name="report_type" value="illegal" id="illegal" required> <label for="illegal">{{ call $.T "illegal_content" }}</label><br />
-                    <input type="radio" name="report_type" value="spam" id="spam" required> <label for="spam">{{ call $.T "spam_garbage" }}</label><br />
-                    <input type="radio" name="report_type" value="wrongcat" id="wrongcat" required> <label for="wrongcat">{{ call $.T "wrong_category" }}</label><br />
-                    <input type="radio" name="report_type" value="dup" id="dup" required> <label for="dup">{{ call $.T "duplicate_deprecated" }}</label><br />
+                    <input type="radio" name="report_type" value="illegal_content" id="illegal" required> <label for="illegal">{{ call $.T "illegal_content" }}</label><br />
+                    <input type="radio" name="report_type" value="spam_garbage" id="spam" required> <label for="spam">{{ call $.T "spam_garbage" }}</label><br />
```

Comment:
We don't need that :). illegal_content is a string in the translation. So why would you call getReportDescription? 
getReportDescription has one and only use: the report listing on torrent_report.html. Why? Because the value of the old reports are not the same as the one in this form. So when you try to call $.T, it can't translate those. However, getReportDescription will change the old values